webapp/channels/src/components/app_bar/app_bar.scss

```diff
@@ -122,8 +122,8 @@ $app-bar-width: 48px;
         .app-bar__divider {
             width: 28px;
             border-top: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
-            margin-top: 14px;
-            margin-bottom: 14px;
+            margin-top: 0px;
+            margin-bottom: 16px;
```

Comment:
Moved to 16 since it is the space that every button is putting after.

Reply:
Always nice to find there's another similar-but-separate bug from the one you're fixing 😛